File: C/lt_01_estseq/lt01_estseq_37.c
#include <stdio.h>

void fibbo(int *num1, int *num2, int *num) {
  if (*num > 0) {
    *num1 += *num2;
    *num -= 1;
    printf("%d\n", *num);
    fibbo(num2, num1, num);
  }
}

int fibboStr(int num) {
  int num1 = 0, num2 = 1;
  if (num > 2) {
    num--;
    fibbo(&num1, &num2, &num);
    if (num1 > num2) {
      return num1;
    } else {
      return num2;
    }
  } else if (num <= 2) {
    return num2;
  } else {
    return 0;
  }
}

int main() {
  int num;

  printf("Insira o nesimo termo: ");
  scanf("%d", &num);

  printf("O %d° termo da Serie de Fibonnaci e %d.\n", num, fibboStr(num));
  return 0;
}

File: Python/lt_01_estseq/lt01_estseq_37.py
nesimo = int(input("Insira o N'ésimo termo a ser encontrado: "))
Vet = [1, 1, 1]
if nesimo > 2:
    for i in range(3, nesimo + 1, 1):
        Vet[2] = Vet[0] + Vet[1]
        Vet[0] = Vet[1]
        Vet[1] = Vet[2]

print("O valor do Termo", nesimo, "é", Vet[2])
